pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2543,17 +2543,19 @@
       ],
       "InboxOn": {
         "netcoreapp2.1": "4.1.0.0",
+        "netcoreapp3.0": "4.2.0.0",
         "monoandroid10": "Any",
         "monotouch10": "Any",
-        "uap10.0.16300": "4.1.0.0",
+        "uap10.0.16300": "4.2.0.0",
         "xamarinios10": "Any",
         "xamarinmac20": "Any",
         "xamarintvos10": "Any",
         "xamarinwatchos10": "Any"
       },
       "AssemblyVersionInPackageVersion": {
         "4.0.1.0": "4.5.0",
-        "4.1.0.0": "4.6.0"
+        "4.1.0.0": "4.6.0",
+        "4.2.0.0": "4.7.0"
```

Comment:
We shouldn't even be building the System.Memory package so I wouldn't have expected this to change.

Reply:
I believe it complained about this, but I may be remembering incorrectly. I'll remove it if I can.